cppForSwig/Makefile.am

```diff
@@ -0,0 +1,120 @@
+LINK = $(CXX)
+
+PYTHON_INCLUDES=@PYTHON_INCLUDES@
+PYTHON_LIBS=@PYTHON_LIBS@
+
+if TARGET_LINUX
+UNAME=Linux
+else
+if TARGET_WINDOWS
+UNAME=Windows
+else
+if TARGET_DARWIN
+UNAME=Darwin
+endif
+endif
+endif
+
+# Export variables to make them available to cryptopp Makefile.
+export AR
+export CXX
+export RANLIB
+export UNAME
+
+if TARGET_LINUX
+CRYPTOPP_EXT=.a
+else
+if TARGET_WINDOWS
+CRYPTOPP_EXT=.a
```

Comment:
Yeah, I suppose that would be more efficient. Just hold off on making changes until I've given everything a spin and have posted a note back here.